bpf/node_config.h

```diff
@@ -176,6 +176,7 @@ DEFINE_IPV6(HOST_IP, 0xbe, 0xef, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0xa, 0x
 #define LB4_MAGLEV_MAP_OUTER test_cilium_lb4_maglev_outer
 #define LB6_MAGLEV_MAP_OUTER test_cilium_lb6_maglev_outer
 #define LB4_SKIP_MAP test_cilium_skip_lb4
+#define LB6_SKIP_MAP test_cilium_skip_lb6
```

Comment:
> * ⚠️ conflict due to `LB6_SKIP_MAP` not being present on `v1.16`. Resolved by dropping the respective hunk of the patch (i.e. [71b9cde](https://github.com/cilium/cilium/commit/71b9cdef49bc42688535f98ec59e9dc09ce5c6b8)).

It's actually there, but it's hidden in `bpf_sock.c` instead of `lib/lb.h`.